docs/User/Troubleshooting.md

```diff
@@ -26,41 +38,39 @@ Error code: 6002, as shown in the following screenshot occurs when Swarm Learnin
 3.  Verify if the Swarm licenses are installed using APLS web management console. For more information, see APLS User Guide.
 
 
-## Installation of HPE Swarm Learning on air-gaped systems or if the Web UI Installer runs into any issue and not able to install
+## 2. Installation of HPE Swarm Learning on air-gaped systems or if the Web UI Installer runs into any issue and not able to install
```

Comment:
Old content.

Need to pick this from Release notes.pdf ( Page 5)

Reply:
"hpe_eval"  path in the docker pull command has changed to "hpe"   Search for this string in the documentation.   We should not have it.